tests/contrib/hooks/test_aws_lambda_hook.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import unittest
+import io
+import json
+import zipfile
+import base64
+
+from airflow.contrib.hooks.aws_lambda_hook import AwsLambdaHook
+
+try:
+    from moto import mock_lambda
+except ImportError:
+    mock_lambda = None
+
+
+class TestAwsLambdaHook(unittest.TestCase):
+
+    @unittest.skipIf(mock_lambda is None, 'mock_dynamodb2 package not present')
+    @mock_lambda
+    def test_get_conn_returns_a_boto3_connection(self):
+        hook = AwsLambdaHook(aws_conn_id='aws_default',
+                             function_name="test_function")
+        self.assertIsNotNone(hook.get_conn())
+
+    def lambda_function(self):
+        code = """
+def lambda_handler(event, context):
```

Comment:
can we textwrap.dedent here instead

Reply:
thanks @saguziel for reviewing my PR :) Pushed the change.